radio/src/opentx.h

```diff
@@ -1068,6 +1068,16 @@ inline void getMixSrcRange(const int source, int16_t & valMin, int16_t & valMax,
 }
 #endif
 
+#if defined(CPUARM) && defined(GVARS)
```

Comment:
Minor quibble, but could we combine this with the `#if defined(CPUARM)` just above?  Also that block needs a closing comment. `#endif   // defined(CPUARM)`

Also technically you only care if `GVAR_MAX` is defined, not `GVARS` per-se.  If the function is unused it will get optimized out anyway.  I realize one probably won't exist w/out the other anyway, so it's a minor point again.

Otherwise looks good to me!